src/lib/draw/route/RouteControls.svelte

```diff
@@ -1,59 +1,75 @@
 <script lang="ts">
-  import { CollapsibleCard } from "lib/common";
-  import {
-    Checkbox,
-    CheckboxGroup,
-    DefaultButton,
-    SecondaryButton,
-  } from "lib/govuk";
-  import { userSettings } from "stores";
-  import { RouteTool } from "./route_tool";
+  import { Checkbox, CheckboxGroup } from "lib/govuk";
+  import { routeTool, userSettings } from "stores";
 
-  export let routeTool: RouteTool;
   // Start with this enabled or disabled, based on whether we're drawing a new
   // route or editing an existing.
   export let extendRoute: boolean;
 
   // TODO When editing, we should save in the route and use the previous value
-  $: routeTool.setRouteConfig({
+  $: $routeTool!.setRouteConfig({
     avoid_doubling_back: $userSettings.avoidDoublingBack,
     extend_route: extendRoute,
   });
 
-  // TODO Show if shift is held or not
-  // TODO Disable finish when the route is invalid
+  // TODO It'd be nice to wire up reactivity to RouteTool directly.
```

Comment:
In Svelte 5, this will be possible. In the meantime, I am thinking about rewriting the 3 .ts tools as components, to make use of reactivity directly and to not dodge around svelte-maplibre